propagation/http/http_test.go

```diff
@@ -0,0 +1,326 @@
+// Copyright 2019 The OpenZipkin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package http_test
+
+import (
+	stdHTTP "net/http"
+	"testing"
+
+	"github.com/openzipkin-contrib/zipkin-go-opentracing/propagation/http"
+
+	"github.com/opentracing/opentracing-go"
+	zipkin "github.com/openzipkin/zipkin-go"
+	"github.com/openzipkin/zipkin-go/model"
+	zb3 "github.com/openzipkin/zipkin-go/propagation/b3"
+	"github.com/openzipkin/zipkin-go/reporter/recorder"
+)
+
+func TestHTTPExtractFlagsOnly(t *testing.T) {
```

Comment:
a lot of this feels like copy/paste. is there a way to only test what's relevant here (employing the code from zipkin-go and only addressing wrapping concerns)

Reply:
unfortunately we reuse a very simple thing from zipkin-go in this package but makes sense to revisit the test list.